### PR TITLE
Feature/add k8s jar

### PR DESCRIPTION
**Pull request by pyang55**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -6,6 +6,9 @@ RUN mkdir /usr/local/logicmonitor
 # NTP is needed for some collector operations
 RUN apt-get update && apt-get install -y ntp
 
+# download kubernetes jar and place into temp directory to be moved after collector installation
+ADD https://static-prod.logicmonitor.com/logicmonitor-k8s/kubernetes.jar /tmp/
```
Comment (pyang55) [inline Dockerfile]: tried creating the entire directory "/usr/local/logicmonitor/agent/lib/amazonlibs, this screws up collector creation from the sdk so moving to tmp

File: startup.py [hunk 2]
```diff
@@ -36,7 +37,7 @@ def startup(params):
     collector = Collector(params)
 
     # detect whether collector already exists
-    if os.path.isdir(install_dir):
+    if glob.glob('/usr/local/logicmonitor/logicmonitorsetup\d+_\d+.bin'):
```
Comment (pyang55) [inline startup.py]: we should be checking for existence of a logicmonitor collector bin, not just a directory existence

File: startup.sh [hunk 3]
```diff
@@ -22,6 +22,7 @@ pid="$!"
 while true
 do
   if [ -e "/usr/local/logicmonitor/agent/logs/wrapper.log" ]; then
+    cp /tmp/*.jar /usr/local/logicmonitor/agent/lib/amazonlibs/
```
Comment (pyang55) [inline startup.sh]: this just copies the kubernetes jar to the right libs directory

Comment (woz5999) [inline startup.sh]: You should skip this step and just add the jar to this directory in the dockerfile. The improved installed collector detection will allow you to do so without this workaround.

**Pull request merged**